geode-core/src/main/java/org/apache/geode/cache/AttributesFactory.java

```diff
@@ -193,7 +193,7 @@
  * others will only read. <br>
  * {@link #setConcurrencyLevel} {@link RegionAttributes#getConcurrencyLevel}</dd>
  *
- * <dt>ConcurrencyChecksEnabled [<em>default:</em> {@code false}]</dt>
+ * <dt>ConcurrencyChecksEnabled [<em>default:</em> {@code true}]</dt>
```

Comment:
how about adding:
      assertTrue(attrs.getConcurrencyChecksEnabled());
to: org.apache.geode.cache.AttributesFactoryJUnitTest.testDefaultConfiguration()

Reply:
I've done this & the test passes.  I'm not going to push the change though - it doesn't warrant a new precheckin run.